src/app/app-routing.module.ts

```diff
@@ -1,8 +1,14 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
+import { LandingPageComponent } from './components/landing-page/landing-page.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '',
+    component: LandingPageComponent
```

Comment:
We want to display content on the homepage. Otherwise we end up with 2 navbars as the `LandingPage` contains the nav bar.

Maybe an advanced `Search` feature that gives the user the option to search via resource (people, ships, planets, etc). What do you think?

Reply:
The test seems to be testing more the router functionality vs checking what's being rendered.

Reply:
Im going to freeze this PR for now, so move onto a different task until we've hashed this out. I think your team can take on the task of creating an `AdvancedSearch` bar to display on the home route (`/` or `" "`). Let me know what you guys think after lunch. Until then, you can continue work on a different task.